CONFIGURATION.md

```diff
@@ -0,0 +1,145 @@
+## Blackbox exporter configuration
+
+### Module
+```yml
+
+  # The protocol over which the probe will take place
+  [ prober: <prober_value> ]
```

Comment:
The types should be defined, similar to how Prometheus does it. The valid values of this particular field should also be stated